app/services/ccr/fee/basic_fee_adapter.rb

```diff
@@ -24,42 +24,27 @@
 #
 module CCR
   module Fee
-    class BasicFeeAdapter < BaseFeeAdapter
-      BASIC_FEE_BILL_MAPPINGS = {
-        GRRAK: zip(%w[AGFS_FEE AGFS_FEE]), # Cracked Trial - LGFS only
-        GRCBR: zip(%w[AGFS_FEE AGFS_FEE]), # Cracked before retrial - LGFS only
-        GRDIS: zip(%w[AGFS_FEE AGFS_FEE]), # Discontinuance
-        GRGLT: zip(%w[AGFS_FEE AGFS_FEE]), # Guilty plea
-        GRRTR: zip(%w[AGFS_FEE AGFS_FEE]), # Retrial
-        GRTRL: zip(%w[AGFS_FEE AGFS_FEE]) # Trial
-      }.freeze
+    class BasicFeeAdapter < SimpleBillAdapter
+      acts_as_simple_bill bill_type: 'AGFS_FEE', bill_subtype: 'AGFS_FEE'
 
       def claimed?
-        maps? && charges?
+        charges?
```

Comment:
**[nit pick/not a blocker]** Since the method just calls another method, consider using:

```ruby
alias claimed? charges?
```

or 

```ruby
alias_method :claimed?, :charges?
```

instead

Reply:
👍 am going to simply this by removing the `charges?` method altogether